### PR TITLE
Translate to Spanish the index section of the basic training

### PR DESCRIPTION
**Pull request by pabloaledo**

### CODE REVIEW

**Comment by mribeirodantas** [comment]
Yay! Very nice to see you here, @pabloaledo 🥳. In Portuguese, I have been merging PRs whenever there is at least one positive review from another Portuguese translator. In Spanish, I think you and @atrigila could try to do this. I can try to review Spanish translations, sure, but I'm not that good so that could be risky 😬 @atrigila Do you think you can help with that? Hopefully there'll be more Spanish translators soon 🤩

File: docs/basic_training/index.es.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+description: Descripción general del material de formación básico de Nextflow
+hide:
+    - toc
+---
+
+# Bienvenido
+
+## Bienvenido al taller de capacitación de Nextflow
+
+Nos complace acompañarlo en el camino para escribir flujos de trabajo científicos reproducibles y escalables usando Nextflow. Esta guía complementa la documentación completa de Nextflow; si alguna vez tiene alguna duda, diríjase a los documentos que se encuentran [aquí] (https://www.nextflow.io/docs/latest).
```
Comment (atrigila) [inline docs/basic_training/index.es.md:11]: this section  [aquí] (https://www.nextflow.io/docs/latest) is not rendering properly

File: docs/basic_training/index.es.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Al final de este curso usted debe:
```
Comment (atrigila) [inline docs/basic_training/index.es.md:13]: Al final de este curso usted _**debería**_

File: docs/basic_training/index.es.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+2. Conocer los conceptos básicos de Nextflow de Canales, Procesos y Operadores
+3. Comprender los flujos de trabajo en contenedores
+4. Comprender las diferentes plataformas de ejecución compatibles con Nextflow
+5. Conocer la comunidad y el ecosistema de Nextflow
+
+## Descripción general
+
+Para que comience con Nextflow lo más rápido posible, seguiremos los siguientes pasos:
+
+1. Configure un entorno de desarrollo para ejecutar Nextflow
```
Comment (atrigila) [inline docs/basic_training/index.es.md:25]: Como la oración anterior dice "seguiremos los siguientes pasos" me parece que los verbos deberían conjugarse en infinitivo como "**configurar** un entorno..." **explorar** los conceptos..."

File: docs/basic_training/index.es.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+2. Explore los conceptos de Nextflow utilizando algunos flujos de trabajo básicos, incluido un análisis de RNA-Seq en varios pasos
+3. Cree y use contenedores Docker para encapsular todas las dependencias del flujo de trabajo
+4. Profundice en la sintaxis central de Nextflow, incluidos canales, procesos y operadores
+5. Pruebe escenarios de implementación en la nube y explore las capacidades de Nextflow Tower
+
+Esto le dará una amplia comprensión de Nextflow, para comenzar a escribir sus propios programas. ¡Esperamos que disfrute del curso! Este es un documento en constante evolución: los comentarios siempre son bienvenidos.
```
Comment (atrigila) [inline docs/basic_training/index.es.md:31]: I would replace _programas_ for _pipelines_ or _flujos de trabajo_

**Comment by atrigila** [comment]
Sure @mribeirodantas, count me in! Thank you very much @pabloaledo  for contributing. I reviewed this translation and these are my suggestions:

- Perhaps we should agree beforehand what kind of tone we would like to give to the material. Unlike English, we can either talk to the reader in **tú** (informal) or in **usted** (formal). The previous material I translated I've adressed the reader as tú, but here it is adressed as usted. 
- There is a section that is not rendering properly in the website. I think there is an extra space in the markdown.

**Comment by mribeirodantas** [comment]
Very good point, @atrigila. I have an open PR with a note about translations to Portuguese (check [here](https://github.com/nextflow-io/training/pull/185/files)). Feel free to add notes to Spanish in this file too, so that people can have a guideline on how to translate the content. I created a translation glossary too for Portuguese. I think it's a good idea to have one for Spanish too! :)

**Comment by pabloaledo** [comment]
Thanks a lot for yout comments.

Regarding the treatment of the reader, I've switched to the "informal" talk (tú). It sounds a bit off for me, but I guess that's ok :). Can you @mribeirodantas confirm if we want to stay like that?.

**Comment by mribeirodantas** [comment]
I think informal is fine, as this is community training material.

**Comment by mribeirodantas** [comment]
This is ready to merge, right @pabloaledo ?

**Comment by pabloaledo** [comment]
> This is ready to merge, right @pabloaledo ?

yes it is :slightly_smiling_face:

**Pull request merged**